components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -1,1130 +1,25 @@
-    clp_config: CLPConfig,
-    container_clp_config: CLPConfig,
-    num_cpus: int,
-    mounts: CLPDockerMounts,
-):
-    celery_method = "job_orchestration.executor.compress"
-    celery_route = SchedulerType.COMPRESSION
-    compression_worker_mounts = [mounts.archives_output_dir]
-    generic_start_worker(
-        COMPRESSION_WORKER_COMPONENT_NAME,
-        instance_id,
-        clp_config,
-        clp_config.compression_worker,
-        container_clp_config,
-        celery_method,
-        celery_route,
-        clp_config.redis.compression_backend_database,
-        num_cpus,
-        mounts,
-        compression_worker_mounts,
-    )
-
-
-def start_query_worker(
-    instance_id: str,
-    clp_config: CLPConfig,
-    container_clp_config: CLPConfig,
-    num_cpus: int,
-    mounts: CLPDockerMounts,
-):
-    celery_method = "job_orchestration.executor.query"
-    celery_route = SchedulerType.QUERY
-
-    query_worker_mounts = [mounts.stream_output_dir]
-    if StorageType.FS == clp_config.archive_output.storage.type:
-        query_worker_mounts.append(mounts.archives_output_dir)
-
-    generic_start_worker(
-        QUERY_WORKER_COMPONENT_NAME,
-        instance_id,
-        clp_config,
-        clp_config.query_worker,
-        container_clp_config,
-        celery_method,
-        celery_route,
-        clp_config.redis.query_backend_database,
-        num_cpus,
-        mounts,
-        query_worker_mounts,
-    )
-
-
-def generic_start_worker(
-    component_name: str,
-    instance_id: str,
-    clp_config: CLPConfig,
-    worker_config: BaseModel,
-    container_clp_config: CLPConfig,
-    celery_method: str,
-    celery_route: str,
-    redis_database: int,
-    num_cpus: int,
-    mounts: CLPDockerMounts,
-    worker_specific_mounts: Optional[List[Optional[DockerMount]]],
-):
-    logger.info(f"Starting {component_name}...")
-
-    container_name = f"clp-{component_name}-{instance_id}"
-    if container_exists(container_name):
-        return
-
-    logs_dir = clp_config.logs_directory / component_name
-    logs_dir.mkdir(parents=True, exist_ok=True)
-    container_logs_dir = container_clp_config.logs_directory / component_name
-
-    # Create necessary directories
-    clp_config.archive_output.get_directory().mkdir(parents=True, exist_ok=True)
-    clp_config.stream_output.get_directory().mkdir(parents=True, exist_ok=True)
-
-    clp_site_packages_dir = CONTAINER_CLP_HOME / "lib" / "python3" / "site-packages"
-    container_worker_log_path = container_logs_dir / "worker.log"
-    # fmt: off
-    container_start_cmd = [
-        "docker", "run",
-        "-di",
-        "--network", "host",
-        "-w", str(CONTAINER_CLP_HOME),
-        "--name", container_name,
-        "--log-driver", "local",
-        "-u", f"{os.getuid()}:{os.getgid()}",
-    ]
-    # fmt: on
-
-    env_vars = [
-        *get_common_env_vars_list(include_clp_home_env_var=True),
-        *get_celery_connection_env_vars_list(container_clp_config),
```

Comment:
we forgot to remove `get_celery_connection_env_vars_list` in this PR, which now has no caller any more.

@coderabbitai create an issue to track the removal of get_celery_connection_env_vars_list . if you see any other methods in any Python code that have no caller at all, report those in the same PR as well